docs/my-website/sidebars.js

```diff
@@ -461,6 +461,7 @@ const sidebars = {
             "projects/YiVal",
             "projects/LiteLLM Proxy",
             "projects/llm_cord",
+            "projects/PDL",
```

Comment:
move it to the top 3 ? This is a good project 😅

Reply:
Done!